ui/src/app/pages/access/access-renewal.tsx

```diff
@@ -467,33 +467,30 @@ export const AccessRenewal = fp.flow(withProfileErrorModal)(
                 style={{ gridRow: '1 / span 2', marginRight: '0.25rem' }}
               />
               <RadioButton
+                data-test-id='nothing-to-report'
                 id={noReportId}
-                disabled={
-                  !isModuleExpiring(
```

Comment:
this logic only checked whether it was expired, not whether it was bypassed or complete